<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>961</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QGroupBox" name="entryGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Entry</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>931</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="3" column="5">
         <widget class="QPushButton" name="clearPushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>trash48.png</normaloff>trash48.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QPushButton" name="submitPushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>checkmark48.png</normaloff>checkmark48.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QLabel" name="fnamelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QLabel" name="mnamelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Middle Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" rowspan="2">
         <widget class="QLabel" name="lnamelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" rowspan="2">
         <widget class="QLabel" name="genderlabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Sex</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" rowspan="2">
         <widget class="QLabel" name="agelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="fnameLineEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="mnameLineEdit"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lnameLineEdit"/>
        </item>
        <item row="3" column="3">
         <widget class="QComboBox" name="sexComboBox"/>
        </item>
        <item row="3" column="4">
         <widget class="QSpinBox" name="ageSpinBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Records">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>343</height>
       </size>
      </property>
      <property name="title">
       <string>Records</string>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>931</width>
         <height>301</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>lnameLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>mnameLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>884</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>fnameLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>874</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
